bin/cli.js

```diff
@@ -71,7 +71,12 @@ For more information, see https://webpack.js.org/api/cli/.`);
 			options = require("./utils/convert-argv")(argv);
 		} catch (err) {
 			if (err.name !== "ValidationError") {
-				throw err;
+				console.error("\n\u001b[1m\u001b[31mWebpack not found, please install webpack");
```

Comment:
The message is misleading. What if you have actually installed webpack?

Reply:
So the behaviour is something like this.
_In both cases, webpack-cli has been installed._

1. **When you haven't installed webpack & no args passed to the CLI.**

The output is an error which is currently not been handled. :arrow_down: 

![Screenshot from 2019-04-26 18-03-46](https://user-images.githubusercontent.com/32242596/56807957-e5e0db80-684d-11e9-8fd1-187dc0018205.png)

2. **When webpack is installed, but no args passed to the CLI.**

After installing webpack, if you run webpack-cli with no args. It shows this message :arrow_down: 

![Screenshot from 2019-04-26 18-04-18](https://user-images.githubusercontent.com/32242596/56808051-348e7580-684e-11e9-84fe-0f5a8e095bf1.png)

#### Pls, acknowledge if you can able to achieve similar behaviour.

Reply:
@pranshuchittora the last line error message in the second image about the `./src` , Is it genuine?
I mean does your config file really have `./src` mentioned in the resolve? Cause I am getting that error even though I didn't mention that path in my config file. PTAL